src/com/exadel/bootcamp/CommandExecutor.java

```diff
@@ -0,0 +1,167 @@
+package com.exadel.bootcamp;
+
+import java.util.Scanner;
+
+public class CommandExecutor {
+    public static void identifyUser() {
+        System.out.println("Welcome, enter you name");
+        Scanner scanner = new Scanner(System.in);
+        try {
+            while (scanner.hasNextLine()){
+                String command = scanner.nextLine();
+                if (command.equals("quit")) {
+                    scanner.close();
+                }
+                else if (command.equals("admin")) {
+                    runAdmin();
+                } else {
+                    runClient();
+                }
+
+                System.out.println("command set: " + command);
+            }
+        } catch(Exception e){
+            System.out.println("Error ::"+e.getMessage());
+            e.printStackTrace();
+        } finally {
+            if (scanner != null)
+                scanner.close();
+        }
+    }
+
+//    public static void identifyUser() {
```

Comment:
(cosmetic) It is a bad practice to leave commented-out code in a PR, when you are on a "serious" project. It confuses a reviewer)